demonstrations_v2/tutorial_stabilizer_codes/demo.py

```diff
@@ -0,0 +1,781 @@
+r"""
+Stabilizer codes for quantum error correction
+=================================================
+
+State-of-the-art quantum devices, such as IBM's Condor and Atom Computing's optical lattices, contain more than
+a thousand qubits. Does this qubit count suffice for valuable quantum algorithms with clear speedups?
+The reality is that there is more to the story than the sheer number of qubits. As we currently stand, quantum
+devices are still prone to errors that increase with device size. For this reason, **quantum error correction**--one of the most important domain in the universe of quantum computing--has
+been gaining traction.
+
+Quantum error correction is implemented through **error correction codes,** which come in many varieties that address different error types.
+Is there a unified way to understand all these codes? The **stabilizer formalism** provides such a framework for
+a large class of quantum error correction codes [#Gottesman1997]_. The so-called
+**stabilizer codes**, such as the repetition, Shor, Steane, and surface codes,
+all fall under this formalism.
+
+In this demo, we will introduce the stabilizer formalism using bottom-up approach. We build
+some well-known codes using **stabilizer generators,** from which
+the other elements of the code (codewords, syndrome measurements, etc.) can be reconstructed. Then, we
+represent these codes as quantum circuits and implement them in PennyLane.
+
+.. figure:: ../_static/demonstration_assets/stabilizer_codes/pennylane-demo-stabilizer-codes-large-thumbnail.png
```

Comment:
historically, we've used the OG image here since it provides a white background.

```suggestion
.. figure:: ../_static/demo_thumbnails/opengraph_demo_thumbnails/pennylane-demo-stabilizer-codes-open-graph.png
```